Tests/FTPropertyWrappersTests/KeychainTests.swift

```diff
@@ -1,87 +1,102 @@
 import XCTest
 @testable import FTPropertyWrappers
 
-struct KeychainStorageTestStruct {
-    @KeychainStore(key: "tester.number") var number: Int?
-    @KeychainStore(key: "tester.number") var collection: [Int]?
-}
-
 final class KeychainTests: XCTestCase {
-    func testKeychainBuiltin() {
-        let tester = KeychainStorageTestStruct()
-        XCTAssertNil(tester.number)
-        tester.number = 15
-        XCTAssertEqual(tester.number, 15)
 
-        let tester2 = KeychainStorageTestStruct()
-        XCTAssertEqual(tester2.number, 15)
-        tester2.number = 30
-        XCTAssertEqual(tester.number, 30)
-        XCTAssertEqual(tester2.number, 30)
+    /*
+     *  Notice: Accesible and AccessControl are not being tested in unit tests.
+     *  Those features are implemented in example apps.
+     */
+
+    func testAttribute_kSecAttrDescription() {
```

Comment:
This looks very incomplete. Can we add at least some TO-DO or add this in separate PR?

Reply:
It is :) I've continued preparing tests to avoid idle time :)

Reply:
I am not going to merge uncomplete code. What is the plan with it?